pghoard/restore.py

```diff
@@ -182,7 +182,7 @@ def list_basebackups_http(self, arg):
     def _get_object_storage(self, site, pgdata):
         storage_config = common.get_object_storage_config(self.config, site)
         storage = get_transfer(storage_config)
-        return ObjectStore(storage, self.config["path_prefix"], site, pgdata)
+        return ObjectStore(storage, self.config["backup_sites"][site]["prefix"], site, pgdata)
```

Comment:
this should not have the `site` arg as it already is a part of the prefix

Reply:
This isn't joining directories, we use `site` for something else in `ObjectStore()`

(actualy looks like the only case is printing out the name, using an old style format string..)